### PR TITLE
Feature weights flops

### PR DESCRIPTION
**Pull request by wvangijseghem**
# Pull Request Checklist
 * Merge in the latest Develop branch changes to your branch
 * Remove .pyc files from your repository:
   * Linux/Mac
     * find . -name *.pyc -delete
   * Windows
     * del /S *.pyc
 * Run automatic regression and makes sure everything is passing
 * Did you make a new regression test that covers your new code?
 * Did you update your docstrings?
 * Did you update your headers to include your name and date?
 * Do a final compare with the Develop branch to be sure what you're changing

### CODE REVIEW

File: regression/scripts/Vehicles/Concorde.py [hunk 1]
```diff
@@ -360,6 +362,7 @@ def vehicle_setup():
     
     # setup
     turbojet.number_of_engines = 4.0
+    turbojet.wing_mounted = [True] * int(turbojet.number_of_engines)
```
Comment (timdmacdo) [inline regression/scripts/Vehicles/Concorde.py:365]: Since this is a default, we should find a way to avoid needed to modify it for the number of engines unless there's a case where only some engines are wing mounted.

File: trunk/SUAVE/Input_Output/Results/plot_weight_comparison.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+
+def plot_weight_comparison(lst_files, caseNames=[], units='kg'):
+    if len(caseNames) < len(lst_files):
+        warnings.warn('caseNames and the number of files do not match, adding generic case names')
+        for i in range(len(caseNames), len(lst_files)):
+            caseNames.append("Case " + str(i))
+    if not isinstance(lst_files, list):
+        lst_files = [lst_files]
+    main_weight = {}
```
Comment (timdmacdo) [inline trunk/SUAVE/Input_Output/Results/plot_weight_comparison.py:16]: There are many instances of this but I'll just leave one comment here. In general we try to keep variable assignment in SUAVE in table format, so this should be modified so that the equal signs line up.

File: trunk/SUAVE/Input_Output/Results/plot_weight_comparison.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import warnings
+from SUAVE.Core import Units
+from matplotlib.pyplot import cm
+
+
+def plot_weight_comparison(lst_files, caseNames=[], units='kg'):
+    if len(caseNames) < len(lst_files):
```
Comment (timdmacdo) [inline trunk/SUAVE/Input_Output/Results/plot_weight_comparison.py:10]: The functions need the standard SUAVE headers

File: trunk/SUAVE/Methods/Weights/Correlations/Common/landing_gear.py [hunk 4]
```diff
@@ -29,9 +31,11 @@ def landing_gear(TOW, landing_gear_wt_factor = 0.04):
...
-    
-    return weight
\ No newline at end of file
+    """
+
+    # process
+    weight = landing_gear_wt_factor * vehicle.mass_properties.max_takeoff
+    output = Data()
+    output.main = weight
+    output.nose = 0
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/Common/landing_gear.py:40]: Why is the nose gear 0?

File: trunk/SUAVE/Methods/Weights/Correlations/Common/wing_main_update.py [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+import numpy as np
+from SUAVE.Attributes.Solids.Aluminum import Aluminum
+
+
+# ----------------------------------------------------------------------
+#   Wing Main
+# ----------------------------------------------------------------------
+
+## @ingroup Methods-Weights-Correlations-Common
+def wing_main_update(vehicle, wing):
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/Common/wing_main_update.py:23]: Function name here is confusing.

File: trunk/SUAVE/Methods/Weights/Correlations/FLOPS/fuselage.py [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+        Assumptions:
+            NFUSE = 1, only one fuselage, please change the variable appropriately for multiple fuselages
+            delta_isa = 0, for pressure calculations
+
+        Source:
+            The Flight Optimization System Weight Estimation Method
+
+        Inputs:
+            vehicle - data dictionary with vehicle properties                   [dimensionless]
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/FLOPS/fuselage.py:25]: This should include the values used.

File: trunk/SUAVE/Methods/Weights/Correlations/FLOPS/operating_items.py [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+    if vehicle.passengers < 51:
+        NSTU = 1  # number of flight attendants
+    else:
+        NSTU = 1 + np.floor(vehicle.passengers / 40.)
+
+    WSTUAB = NSTU * 155 + NGALC * 200  # flight attendant weight
+    WFLCRB = NFLCR * 225  # flight crew and baggage weight
+
+    output = Data()
+    output.oper_items = WUF * Units.lbs + WOIL * Units.lbs + WSRV * Units.lbs + WCON * Units.lbs
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/FLOPS/operating_items.py:86]: Any data structures passed outside of a file should have verbose names, such as "operating_items" here.

File: trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/operating_items.py [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+        - unusable fuel
+        - engine oil
+        - passenger service
+        - ammunition and non-fixed weapons
+        - cargo containers
+
+        Assumptions:
+
+        Source:
+            http://aerodesign.stanford.edu/aircraftdesign/AircraftDesign.html
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/operating_items.py:29]: Looks like the wrong source here.

File: regression/scripts/Vehicles/Concorde.py (hunk 1 above)
Comment (wvangijseghem) [reply regression/scripts/Vehicles/Concorde.py:365]: I could just set the default argument to True, meaning all engines are mounted on the wing, and if this is not the case then the user can input a list of booleans, indicating which engines are not. Do you think that's a good idea?

File: trunk/SUAVE/Methods/Weights/Correlations/Common/wing_main_update.py (hunk 5 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/Common/wing_main_update.py:23]: FINC

File: regression/scripts/Vehicles/Concorde.py (hunk 1 above)
Comment (wvangijseghem) [reply regression/scripts/Vehicles/Concorde.py:365]: FINC

File: trunk/SUAVE/Input_Output/Results/plot_weight_comparison.py (hunk 3 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Input_Output/Results/plot_weight_comparison.py:10]: FINC

File: trunk/SUAVE/Methods/Weights/Correlations/FLOPS/fuselage.py (hunk 6 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/FLOPS/fuselage.py:25]: FINC

File: trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/operating_items.py (hunk 8 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/operating_items.py:29]: I am not sure what you mean. These operating items weights are taken from Kroo's method presented on http://aerodesign.stanford.edu/aircraftdesign/AircraftDesign.html

File: trunk/SUAVE/Methods/Weights/Correlations/FLOPS/operating_items.py (hunk 7 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/FLOPS/operating_items.py:86]: FINC

File: trunk/SUAVE/Methods/Weights/Correlations/BWB/empty.py [hunk 9]
```diff
@@ -75,139 +78,156 @@ def empty(vehicle):
...
-    output.systems_breakdown.hydraulics         = output_2.wt_hyd_pnu     
-    output.systems_breakdown.instruments        = output_2.wt_instruments 
-    output.systems_breakdown.avionics           = output_2.wt_avionics    
-    output.systems_breakdown.optionals          = output_2.wt_opitems     
-    output.systems_breakdown.electrical         = output_2.wt_elec        
-    output.systems_breakdown.air_conditioner    = output_2.wt_ac          
-    output.systems_breakdown.furnish            = output_2.wt_furnish    
-    
-    #define weights components
+    payload = payload_weight(vehicle)
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/BWB/empty.py:140]: Table format

File: trunk/SUAVE/Methods/Weights/Correlations/BWB/empty.py [hunk 10]
```diff
@@ -75,139 +78,156 @@ def empty(vehicle):
...
-    vehicle.fuel                                = fuel
-    vehicle.apu                                 = apu
-    vehicle.hydraulics                          = hydraulics
-    vehicle.optionals                           = optionals
-    vehicle.landing_gear                        = landing_gear_component
-
-    return output
\ No newline at end of file
+    output.systems_breakdown = Data()
+    output.systems_breakdown.control_systems = output_2.wt_flt_ctrl
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/BWB/empty.py:164]: Table format

File: trunk/SUAVE/Methods/Weights/Correlations/FLOPS/prop_system.py [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+    output.wt_starter           = WSTART
+    output.wt_engine_controls   = WEC
+    output.fuel_system          = WFSYS
+    output.nacelle              = WNAC
+    output.wt_eng               = WENG * NENG
+    return output
+
+
+def nacelle_FLOPS(prop):
+    """ Calculates the weight of the nacelle for all engines"""
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/FLOPS/prop_system.py:80]: These functions will all need docstrings with inputs and outputs

File: trunk/SUAVE/Methods/Weights/Correlations/FLOPS/wing_weight.py [hunk 12]
```diff
@@ -0,0 +1,302 @@
...
+            T[i + 1] = wing.Segments[i].thickness_to_chord
+        else:
+            T[i + 1] = wing.thickness_to_chord
+        SWP[i] = np.arctan(np.tan(wing.Segments[i - 1].sweeps.quarter_chord) - (C[i - 1] - C[i]))
+    SWP[-1] = np.arctan(np.tan(wing.Segments[-2].sweeps.quarter_chord) - (C[-2] - C[-1]))
+    return ETA, C, T, SWP
+
+
+def generate_int_stations(NSD, ETA):
+    """ Divides half of the wing in integration stations"""
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/FLOPS/wing_weight.py:234]: Full docstrings here too

File: trunk/SUAVE/Methods/Weights/Correlations/Raymer/prop_system.py [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+    output.wt_thrust_reverser   = WTHR
+    output.wt_starter           = WSTART
+    output.wt_engine_controls   = WEC
+    output.fuel_system          = WFSYS
+    output.nacelle              = WNAC
+    output.wt_eng               = WENG * NENG
+    return output
+
+def nacelle_Raymer(vehicle, prop, WENG):
+    NENG    = prop.number_of_engines
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/Raymer/prop_system.py:69]: Full docstrings

File: trunk/SUAVE/Methods/Weights/Correlations/Raymer/systems.py [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+    WFURN += 75 * NFLCR
+    WFURN += 2.5 * vehicle.passengers**1.33
+
+    Vpr = D ** 2 * np.pi / 4 * L
+    WAC = 62.36 * vehicle.passengers ** 0.25 * (Vpr / 1000) ** 0.604 * Wuav ** 0.1
+
+    WAI = 0.002 * DG
+
+    output                  = Data()
+    output.wt_flt_ctrl      = WSC * Units.lbs
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/Raymer/systems.py:90]: Change to verbose names

File: trunk/SUAVE/Methods/Weights/Correlations/Raymer/wing_main_raymer.py [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+def wing_main_raymer(vehicle, wing):
+    """ Calculate the wing weight of the aircraft based the Raymer method
+
+    Assumptions:
+
+    Source:
+        Aircraft Design: A Conceptual Approach
+
+    Inputs:
+        vehicle - data dictionary with vehicle properties                    [dimensionless]
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/Raymer/wing_main_raymer.py:22]: Expand input description

File: trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/systems.py [hunk 16]
```diff
@@ -47,93 +50,93 @@ def systems(num_seats, ctrl_type, S_h, S_v, S_gross_w, ac_type):
...
-    output.wt_hyd_pnu     = hyd_pnu_wt
-    output.wt_instruments = instruments_wt
-    output.wt_avionics    = avionics_wt
-    output.wt_opitems     = opitems_wt
-    output.wt_elec        = elec_wt
-    output.wt_ac          = ac_wt
-    output.wt_furnish     = furnish_wt
-    output.wt_systems     = output.wt_flt_ctrl + output.wt_apu + output.wt_hyd_pnu \
+    output = Data()
+    output.wt_flt_ctrl      = flt_ctrl_wt
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/systems.py:129]: Use verbose names

Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/systems.py:129]: FINC

File: trunk/SUAVE/Methods/Weights/Correlations/Raymer/wing_main_raymer.py (hunk 15 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/Raymer/wing_main_raymer.py:22]: FINC

File: trunk/SUAVE/Methods/Weights/Correlations/Raymer/systems.py (hunk 14 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/Raymer/systems.py:90]: FINC

File: trunk/SUAVE/Methods/Weights/Correlations/Raymer/prop_system.py (hunk 13 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/Raymer/prop_system.py:69]: FINC

File: trunk/SUAVE/Methods/Weights/Correlations/FLOPS/wing_weight.py (hunk 12 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/FLOPS/wing_weight.py:234]: FINC

File: regression/scripts/Vehicles/Boeing_747.py [hunk 17]
```diff
@@ -21,14 +21,24 @@
...
 def vehicle_setup():
 
     vehicle = SUAVE.Vehicle()
     
     #print vehicle
     vehicle.mass_properties.max_zero_fuel=238780*Units.kg
-    vehicle.mass_properties.max_takeoff  =785000.*Units.lbs
+    vehicle.mass_properties.max_takeoff  =833000.*Units.lbs
+
+    vehicle.design_mach_number = 0.92
```
[the remark this answers is not in the document]
Comment (wvangijseghem) [reply regression/scripts/Vehicles/Boeing_747.py:34]: FINC

File: regression/scripts/Vehicles/Boeing_747.py [hunk 18]
```diff
@@ -149,11 +166,226 @@ def vehicle_setup():
...
+    # size the turbofan
+    turbofan_sizing(turbofan, mach_number, altitude)
+
+    # add  gas turbine network turbofan to the vehicle
+    vehicle.append_component(turbofan)
+
+    # ------------------------------------------------------------------
+    #  Landing Gear
+    # ------------------------------------------------------------------
+    landing_gear = SUAVE.Components.Landing_Gear.Landing_Gear()
```
[the remark this answers is not in the document]
Comment (wvangijseghem) [reply regression/scripts/Vehicles/Boeing_747.py:377]: FINC

File: regression/scripts/Vehicles/Boeing_737.py [hunk 19]
```diff
@@ -45,10 +45,12 @@ def vehicle_setup():
     vehicle.mass_properties.cargo                     = 10000.  * Units.kilogram   
     vehicle.mass_properties.center_of_gravity         = [ 15.30987849,   0.        ,  -0.48023939]
     vehicle.mass_properties.moments_of_inertia.tensor = [[3173074.17, 0 , 28752.77565],[0 , 3019041.443, 0],[0, 0, 5730017.433]] # estimated, not correct 
-    
+    vehicle.design_mach_number = 0.78
```
[the remark this answers is not in the document]
Comment (wvangijseghem) [reply regression/scripts/Vehicles/Boeing_737.py:48]: FINC

File: regression/scripts/Vehicles/Concorde.py [hunk 20]
```diff
@@ -531,7 +533,22 @@ def vehicle_setup():
...
     
     # add  gas turbine network gt_engine to the vehicle
-    vehicle.append_component(turbojet)      
+    vehicle.append_component(turbojet)
+
+    # ------------------------------------------------------------------
+    #  Landing Gear
+    # ------------------------------------------------------------------
+    landing_gear = SUAVE.Components.Landing_Gear.Landing_Gear()
+    landing_gear.tag = "main_landing_gear"
```
[the remark this answers is not in the document]
Comment (wvangijseghem) [reply regression/scripts/Vehicles/Concorde.py:542]: FINC

File: regression/scripts/weights/weights.py [hunk 21]
```diff
@@ -49,29 +49,29 @@ def main():
...
     print('Results (kg)')
     print(weight)
     
     print('Relative Errors')
     print(error)  
       
-    for k,v in list(error.items()):
-        assert(np.abs(v)<1E-6)    
+    for k,v in list(error.items()):
+        assert(np.abs(v)<1)
```
[the remark this answers is not in the document]
Comment (wvangijseghem) [reply regression/scripts/weights/weights.py:74]: Done

File: regression/scripts/weights/weights.py [hunk 22]
```diff
@@ -116,8 +116,8 @@ def main():
     print('Relative Errors')
     print(error)  
 
-    for k,v in list(error.items()):
-        assert(np.abs(v)<1e-6)    
+    for k,v in list(error.items()):
+        assert(np.abs(v)<1)
```
[the remark this answers is not in the document]
Comment (wvangijseghem) [reply regression/scripts/weights/weights.py:120]: Done

File: regression/scripts/weights/weights.py [hunk 23]
```diff
@@ -157,7 +157,7 @@ def main():
     print(error)  
               
     for k,v in list(error.items()):
-        assert(np.abs(v)<1E-6)    
+        assert(np.abs(v)<1)
```
[the remark this answers is not in the document]
Comment (wvangijseghem) [reply regression/scripts/weights/weights.py:160]: Done

File: regression/scripts/weights/weights.py [hunk 24]
```diff
@@ -173,20 +173,19 @@ def main():
...
+    error.vertical_tail = (actual.vertical_tail - weight.vertical_tail) / actual.vertical_tail
     print('Results (kg)')
     print(weight)
     
     print('Relative Errors')
     print(error)  
               
     for k,v in list(error.items()):
-        assert(np.abs(v)<1E-6)    
+        assert(np.abs(v)<1)
```
[the remark this answers is not in the document]
Comment (wvangijseghem) [reply regression/scripts/weights/weights.py:188]: Done

File: trunk/SUAVE/Methods/Weights/Correlations/BWB/empty.py [hunk 25]
```diff
@@ -75,139 +78,156 @@ def empty(vehicle):
...
-    TOW        = vehicle.mass_properties.max_takeoff
-    wt_zf      = vehicle.mass_properties.max_zero_fuel
-    wt_cargo   = vehicle.mass_properties.cargo
-    num_pax    = vehicle.passengers
-    ctrl_type  = vehicle.systems.control
-    ac_type    = vehicle.systems.accessories
-     
-    num_seats                = vehicle.passengers
-    bwb_aft_centerbody_area  = vehicle.fuselages['fuselage_bwb'].aft_centerbody_area
+    Nult = vehicle.envelope.ultimate_load
```
[the remark this answers is not in the document]
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/BWB/empty.py:84]: FINC

File: trunk/SUAVE/Methods/Weights/Correlations/BWB/empty.py (hunk 9 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/BWB/empty.py:140]: FINC

File: trunk/SUAVE/Methods/Weights/Correlations/BWB/empty.py (hunk 10 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/BWB/empty.py:164]: FINC

File: trunk/SUAVE/Methods/Weights/Correlations/FLOPS/prop_system.py (hunk 11 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/FLOPS/prop_system.py:80]: FINC

File: trunk/SUAVE/Methods/Weights/Correlations/Common/landing_gear.py [hunk 26]
```diff
@@ -29,9 +31,11 @@ def landing_gear(TOW, landing_gear_wt_factor = 0.04):
...
-    
-    return weight
\ No newline at end of file
+    """
+
+    # process
+    weight          = landing_gear_wt_factor * vehicle.mass_properties.max_takeoff
+    output          = Data()
+    output.main     = weight
+    output.nose     = 0
```
[the remark this answers is not in the document]
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/Common/landing_gear.py:40]: This is because the method used here combines the weight of the main landing gear and the nose landing gear into one estimate. I wasn't sure what else I could do, we could say the main landing gear is 80% of the weight and the nose landing gear is 20% of the lumped sum (approximately what I found in literature). Let me know what you suggest.

File: trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/operating_items.py (hunk 8 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/operating_items.py:29]: The all caps variables are to be consistent with the FLOPS method. Let me know if you want this to change

File: regression/automatic_regression.py [hunk 27]
```diff
@@ -38,61 +38,61 @@
...
     'scripts/test_input_output/test_xml_read_write.py',
-    'scripts/test_input_output/test_freemind_write.py',    
-    'scripts/variable_cruise_distance/variable_cruise_distance.py', 
-    'scripts/V_n_diagram/V_n_diagram_regression.py',      
-    'scripts/VTOL/test_Multicopter.py',
-    'scripts/VTOL/test_Tiltwing.py',    
+    'scripts/test_input_output/test_freemind_write.py',
+    'scripts/variable_cruise_distance/variable_cruise_distance.py',
+    'scripts/V_n_diagram/V_n_diagram_regression.py',
+    # 'scripts/VTOL/test_Multicopter.py',
```
Comment (timdmacdo) [inline regression/automatic_regression.py:94]: Needs to be re-added

File: regression/scripts/weights/weights.py [hunk 28]
```diff
@@ -19,72 +19,158 @@
...
 
 def main():
-  
-    vehicle = vehicle_setup()    
-    weight = Tube_Wing.empty(vehicle)
-    
-    # regression values    
+
+    vehicle = vehicle_setup()
+    weight = Common.empty_weight(vehicle, method_type = "SUAVE")
```
Comment (timdmacdo) [inline regression/scripts/weights/weights.py:31]: If we're still using the SUAVE method for the below why do the weights here change?

File: regression/scripts/weights/weights.py [hunk 29]
```diff
@@ -19,72 +19,158 @@
...
-      
-    for k,v in list(error.items()):
-        assert(np.abs(v)<1E-6)    
-   
+    print(error)
+
+    for k, v in error.items():
+        assert (np.abs(v) < 1E-6)
+
+    weight = Common.empty_weight(vehicle, method_type="FLOPS Complex")
```
Comment (timdmacdo) [inline regression/scripts/weights/weights.py:75]: FLOPS simple should also be tested

File: trunk/SUAVE/Methods/Geometry/Two_Dimensional/Planform/wing_planform.py [hunk 30]
```diff
@@ -111,8 +111,6 @@ def wing_planform(wing):
         affected_area           = (wing_chord_flap_start + wing_chord_flap_end) * (flap.span_fraction_end- flap.span_fraction_start)*span / 2.          
          
     # update
-    wing.chords.root                = chord_root
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Geometry/Two_Dimensional/Planform/wing_planform.py:114]: Why were these removed?

File: trunk/SUAVE/Methods/Weights/Correlations/Raymer/fuselage.py [hunk 31]
```diff
@@ -0,0 +1,46 @@
...
+
+def fuselage_weight_Raymer(vehicle, fuse):
+    """ Calculate the weight of the fuselage of a transport aircraft based on the Raymer method
+
+        Assumptions:
+            No fuselage mounted landing gear
+            1 cargo door
+
+        Source:
+            Aircraft Design: A Conceptual Approach
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Weights/Correlations/Raymer/fuselage.py:21]: Which edition?

File: trunk/SUAVE/Methods/Weights/Correlations/Raymer/wing_main_raymer.py (hunk 15 above)
Comment (timdmacdo) [reply trunk/SUAVE/Methods/Weights/Correlations/Raymer/wing_main_raymer.py:22]: This one is still missing

File: trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/operating_items.py (hunk 8 above)
Comment (timdmacdo) [reply trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/operating_items.py:29]: I'd suggested writing out the variables if they're general purpose, but it would be helpful to callout their FLOPS equivalent in comments.

File: trunk/SUAVE/Methods/Weights/Correlations/Common/landing_gear.py (hunk 26 above)
[the remark this answers is not in the document]
Comment (timdmacdo) [reply trunk/SUAVE/Methods/Weights/Correlations/Common/landing_gear.py:40]: This is fine for now, we'll see what the larger group thinks

File: trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/operating_items.py (hunk 8 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/operating_items.py:29]: Changed in this file, I suspect you don't want me to change it in the FLOPS files correct?

File: trunk/SUAVE/Methods/Weights/Correlations/Raymer/wing_main_raymer.py (hunk 15 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/Raymer/wing_main_raymer.py:22]: FINC

File: regression/automatic_regression.py (hunk 27 above)
Comment (wvangijseghem) [reply regression/automatic_regression.py:94]: FINC

File: regression/scripts/weights/weights.py (hunk 28 above)
Comment (wvangijseghem) [reply regression/scripts/weights/weights.py:31]: Good question. 
The wing and fuselage weight change due to the change of ultimate load factor from 2.5 to the default of 3.75.
The empty weight also changes due to the restructuring of the empty weight where the operational items are only included in the OEW and not in the EW. The systems weight changed because the operational items are not included in the system weight anymore and both tail weight are dependent on the ultimate load factor and thus also change.

File: regression/scripts/weights/weights.py (hunk 29 above)
Comment (wvangijseghem) [reply regression/scripts/weights/weights.py:75]: FINC

File: trunk/SUAVE/Methods/Geometry/Two_Dimensional/Planform/wing_planform.py (hunk 30 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Geometry/Two_Dimensional/Planform/wing_planform.py:114]: Because to me, tip and root chord are more easily known than taper and aspect ratio, definitely in the case of multi-segmented wings where the definition of taper and aspect ration can lead to ambiguity in the multiple segments. Therefore in my opinion, when running this script, even though the user probably defined tip and root chord, they are updated without the user's knowledge which can lead to the incorrect result. So we could include a flag if the user wants to update the tip and root chord or not.

File: trunk/SUAVE/Methods/Weights/Correlations/Raymer/fuselage.py (hunk 31 above)
Comment (wvangijseghem) [reply trunk/SUAVE/Methods/Weights/Correlations/Raymer/fuselage.py:21]: FINC

File: trunk/SUAVE/Methods/Geometry/Two_Dimensional/Planform/wing_planform.py (hunk 30 above)
Comment (timdmacdo) [reply trunk/SUAVE/Methods/Geometry/Two_Dimensional/Planform/wing_planform.py:114]: Okay, we'll need to discuss this with the larger group.

**Comment by timdmacdo** [comment]
This looks good. You'll need to fix those conflicts and then we can pull it in to the intermediate SUAVE branch. Only other outstanding item is the Doxygen tags.
